spec/lib/soapy_cake/admin_spec.rb

```diff
@@ -175,8 +175,17 @@
 
     describe '#add_blacklist' do
       let(:method) { :add_blacklist }
-      let(:cake_method) { :blacklist }
-      it_behaves_like 'a cake admin method'
+      let(:date) { Date.new(2015, 9, 3) }
+      let(:request) { double('request') }
+
+      it 'adds to the blacklist' do
```

Comment:
Actually we have the `opts` and `cake_opts` variables for that exact scenario, it just got messed up a bit and is not used anymore for some reason. Do you mind fixing that so that we can stick to the `it_behaves_like` schema?